.travis.yml

```diff
@@ -1,15 +1,21 @@
 language: node_js
 sudo: false
+
 git:
   depth: 10
+
 node_js:
-  - "4"
   - "6"
+  - "8"
+  - "10"
+
 install:
-    - npm install
-    - npm install -g codecov
+  - npm install
```

Comment:
wrong indentation before?

Reply:
I noticed some were indented with 2 spaces while others were 4. Typically I indent things with 2 spaces also, so I went ahead and modified them to at least be in uniform with the others. 

There are no problems if I had left it alone. Travis CI, at least, does not enforce an indention rule for their YAML files while others have been known to enforce two space.